dex-it/src/test/scala/com/wavesplatform/it/api/ApiExtensions.scala

```diff
@@ -22,10 +22,10 @@ trait ApiExtensions extends NodeApiExtensions {
   this: MatcherSuiteBase =>
 
   protected def placeAndAwaitAtDex(
-    order: Order,
-    expectedStatus: HttpOrderStatus.Status = Status.Accepted,
-    dex: DexContainer = dex1,
-    isMarketOrder: Boolean = false
+                                    order: Order,
+                                    expectedStatus: HttpOrderStatus.Status = Status.Accepted,
+                                    dex: DexContainer = dex1,
+                                    isMarketOrder: Boolean = false
```

Comment:
Do we really need this? Does the formatter change it?

Reply:
fixed